acceptance-tests/src/test/java/tech/pegasys/signers/dsl/hashicorp/HashicorpNode.java

```diff
@@ -72,15 +73,18 @@ public String getHost() {
     return hashicorpVaultDocker.getIpAddress();
   }
 
-  public String getSigningKeyPath() {
-    return hashicorpVaultDocker.getVaultSigningKeyPath();
-  }
-
   public int getPort() {
     return hashicorpVaultDocker.getPort();
   }
 
   public Optional<SelfSignedCertificate> getServerCertificate() {
     return serverTlsCertificate;
   }
+
+  /* Note: Path should be the "subpath" of the secret - not the full HTTP path.
+  The full HTTP Path will be returned.
+   */
+  public String addSecretsToVault(final Map<String, String> entries, final String path) {
+    return hashicorpVaultDocker.addSecretsToVault(entries, path);
```

Comment:
does it return the full URL? like `http://....//`?

Reply:
may be return a constructed URL? The method name is not very helpful :)

Reply:
now returns void - just adds a secret - its up the to test to construct the actual path (via helper function)